packages/smooth_app/lib/l10n/app_de.arb

```diff
@@ -861,7 +861,7 @@
     "connect_with_us": "Mit uns in Verbindung treten",
     "instagram": "Instagram",
     "instagram_link": "https://instagram.com/open.food.facts",
-    "twitter": "X (formerly Twitter)",
+    "twitter": "𝕏 (ehemals Twitter)",
```

Comment:
Only for this language the X looks funny on my smartphone.

![Screenshot_2023-08-15-19-42-43](https://github.com/openfoodfacts/smooth-app/assets/11576431/02689be8-53e9-46f2-964c-c2dcb3ab2181)